frontend/package.json

```
@@ -15,7 +15,7 @@
     "@angular-material-extensions/password-strength": "^6.0.0",
     "@angular/animations": "^10.2.3",
```

This is the first single comment. 

```suggestion
    "@angular/animations": "^10.2.3",
```